vllm/triton_utils/sample.py

```diff
@@ -0,0 +1,13 @@
+import math
+
+# This is a hardcoded limit in Triton (max block size).
+MAX_TRITON_N_COLS = 131072
+
+
+def get_num_triton_sampler_splits(n_cols: int) -> int:
```

Comment:
Off topic: @Yard1 I feel this should be a general function for all triton kernels instead of just sampler. Do you think it makes sense to rename it to `get_num_triton_input_chunks` so something similar, and use it [here](https://github.com/vllm-project/vllm/blob/main/vllm/model_executor/layers/fused_moe/fused_moe.py#L455) as well?

Reply:
I think that would make sense

Reply:
Just to clarify: is this something you'd like to have addressed in this PR?

Reply:
No it's not necessary. We can merge this PR first.